docs/samples/custom/torchserve/hugging-face-bert-sample.md

```diff
@@ -0,0 +1,121 @@
+# Torchserve custom server with Hugging Face bert sample
+
+## Model archive file creation
+
+[Torchserve Huggingface Transformers Bert example](https://github.com/pytorch/serve/tree/master/examples/Huggingface_Transformers)
+
+## Download and install transformer models
+
+```bash
+pip install transformers
+```
+
+Modify setup_config.json
+
+```json
+{
+ "model_name":"bert-base-uncased",
+ "mode":"sequence_classification",
+ "do_lower_case":"True",
+ "num_labels":"2",
+ "save_mode":"torchscript",
+ "max_length":"150"
+}
+```
+
+Download models
+
+```bash
+python Download_Transformer_models.py
+```
+
+Create a requirements.txt and add `transformers` to it.
+
+Run the below command with all files in place to generate mar file.
+
+```bash
+torch-model-archiver --model-name BERTSeqClassification --version 1.0 --serialized-file Transformer_model/pytorch_model.bin --handler ./Transformer_handler_generalized.py --extra-files"Transformer_model/config.json,./setup_config.json,./Seq_classification_artifacts/index_to_name.json" -r requirements.txt
```

Comment:
```suggestion
torch-model-archiver --model-name BERTSeqClassification --version 1.0 --serialized-file Transformer_model/pytorch_model.bin --handler ./Transformer_handler_generalized.py --extra-files "Transformer_model/config.json,./setup_config.json,./Seq_classification_artifacts/index_to_name.json" -r requirements.txt
```

Reply:
I have uploaded the marfile to gcs bucket, on custom yaml you can use `STORAGE_URI` env to load the model.
```
gsutil ls gs://kfserving-examples/models/torchserve/bert/
gs://kfserving-examples/models/torchserve/bert/BERTSeqClassification.mar
```